docs/cudf/source/user_guide/index.md

```diff
@@ -12,7 +12,7 @@ groupby
 guide-to-udfs
 cupy-interop
 options
-performance-comparisons
+performance_comparisons/index
```

Comment:
Why did we name the folder `performance_comparisons` with an underscore, and the notebook `performance-comparisons.ipynb` with a dash?